variables.tf

```
@@ -17,65 +17,74 @@
 #------------------------#
 # GSuite export settings #
 #------------------------#
-variable admin_user {
+variable "admin_user" {
   description = "The GSuite Admin user to impersonate"
 }
 
-variable api {
+variable "api" {
   description = "The Admin SDK API to sync data from"
+  default     = "reports_v1"
 }
 
-variable applications {
-  description = "The Admin SDK applications to sync data from (space-separated)"
+variable "applications" {
+  description = "The Admin SDK applications to sync data from"
+  type        = list(string)
+  default     = ["admin", "login", "drive", "mobile", "token"]
 }
 
-variable frequency {
+variable "frequency" {
   description = "The crontab entry that controls the sync frequency"
   default     = "*/10 * * * *"
 }
 
-variable service_account {
+variable "service_account" {
   description = "The service account for exporting GSuite data. Needs domain-wide delegation and correct access scopes."
 }
 
-variable project_id {
+variable "project_id" {
+  type        = "string"
   description = "The project to export GSuite data to."
 }
-   
-variable export_filter {
+
+variable "export_filter" {
   description = "The export filter to use in a log export (if any)"
   default     = ""
 }
 
+variable "gsuite_exporter_version" {
+  description = "Version of the gsuite-exporter PyPi package"
+  default     = "0.0.3" # latest version
```

This comment isn't necessary.

```suggestion
  default     = "0.0.3"
```